src/js/pages/services/DeploymentsTab.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import AlertPanel from '../../components/AlertPanel';
+
+var DeploymentsTab = React.createClass({
```

Comment:
I'd say for new components with should follow the new class paradigm. +1 for the updated docs .